binaries/src/bin/vdaf_message_sizes.rs

```diff
@@ -45,21 +82,20 @@ fn main() {
         prio3_input_share_size(&prio3.shard(&measurement, &nonce).unwrap().1)
     );
 
-    let len = 1000;
-    let prio3 = Prio3::new_sum_vec(num_shares, 1, len).unwrap();
-    let measurement = vec![0; len];
+    //for len in [10, 100, 1000, 10_000, 100_000] {
+    println!("Prio3 SumVec size, single- & multi-threaded");
     println!(
-        "prio3 countvec ({} len) share size = {}",
-        len,
-        prio3_input_share_size(&prio3.shard(&measurement, &nonce).unwrap().1)
-    );
-
-    let prio3 = Prio3::new_sum_vec_multithreaded(num_shares, 1, len).unwrap();
-    println!(
-        "prio3 countvec multithreaded ({} len) share size = {}",
-        len,
-        prio3_input_share_size(&prio3.shard(&measurement, &nonce).unwrap().1)
+        "{:>10} | {:>10} | {:>10} | {:>10}",
+        "Field", "len", "bits", "size"
     );
+    println!("--------------------------------------------------");
+    for len in [100, 1000, 10_000] {
+        for bits in [1, 2, 4, 8, 16, 32, 64] {
+            print_sizes!(Field32, 32, num_shares, len, bits, nonce);
+            print_sizes!(Field64, 64, num_shares, len, bits, nonce);
```

Comment:
It probably makes more sense to add a warning to the actual code that we don't people to use. I'm not sure if we can use [Diagnostic](https://doc.rust-lang.org/proc_macro/struct.Diagnostic.html#method.warning) because it's unstable. So maybe `#[deprecated]`?

Reply:
Field64 is still used in Prio3Count and Poplar1, but we could probably deprecate the fields that don't show up in the VDAF document at all.